GPU/GPUTracking/Base/GPUSettingsList.h

```diff
@@ -47,9 +47,10 @@ AddOptionRTC(TrackReferenceX, float, 1000.f, "", 0, "Transport all tracks to thi
 AddOptionRTC(tpcZSthreshold, float, 2.0f, "", 0, "Zero-Suppression threshold")
 AddOptionRTC(tpcTubeChi2, float, 5.f * 5.f, "", 0, "Max chi2 to mark cluster adjacent to track")
 AddOptionRTC(tpcTubeMaxSize2, float, 2.5f * 2.5f, "", 0, "Square of max tube size (normally derrived from tpcTubeChi2)")
-AddOptionRTC(trdMinTrackPt, float, .5f, "", 0, "Min Pt for tracks to be propagated through the TRD")
```

Comment:
This seems to be a rebasing artifact, the setting should not be deleted.

Reply:
Woops, totally missed that. Just pushed a fix.